web-modeler-openapi-cli/docs/CollaboratorsApi.md

```diff
@@ -0,0 +1,225 @@
+
+<a name="deleteCollaborator"></a>
+
+# **deleteCollaborator**
+
+> deleteCollaborator(projectId, email)
+
+Deletion of resources is recursive and cannot be undone.
+
+### Example
+
+```java
+// Import classes:
+import org.camunda.community.webmodeler.client.invoker.ApiClient;
+import org.camunda.community.webmodeler.client.invoker.ApiException;
+import org.camunda.community.webmodeler.client.invoker.Configuration;
+import org.camunda.community.webmodeler.client.invoker.auth.*;
+import org.camunda.community.webmodeler.client.invoker.models.*;
+import org.camunda.community.webmodeler.client.api.CollaboratorsApi;
+
+public class Example {
+  public static void main(String[] args) {
+    ApiClient defaultClient = Configuration.getDefaultApiClient();
+    defaultClient.setBasePath("https://modeler.cloud.camunda.io");
+    
+    // Configure HTTP bearer authorization: Bearer
+    HttpBearerAuth Bearer = (HttpBearerAuth) defaultClient.getAuthentication("Bearer");
+    Bearer.setBearerToken("BEARER TOKEN");
+
+    CollaboratorsApi apiInstance = new CollaboratorsApi(defaultClient);
+    UUID projectId = new UUID(); // UUID | 
+    String email = "email_example"; // String | 
+    try {
+      apiInstance.deleteCollaborator(projectId, email);
+    } catch (ApiException e) {
+      System.err.println("Exception when calling CollaboratorsApi#deleteCollaborator");
+      System.err.println("Status code: " + e.getCode());
+      System.err.println("Reason: " + e.getResponseBody());
+      System.err.println("Response headers: " + e.getResponseHeaders());
+      e.printStackTrace();
+    }
+  }
+}
+```
+
+### Parameters
+
+|     Name      |      Type       | Description | Notes |
+|---------------|-----------------|-------------|-------|
+| **projectId** | [**UUID**](.md) |             |       |
+| **email**     | **String**      |             |       |
+
+### Return type
+
+null (empty response body)
+
+### Authorization
+
+[Bearer](../README.md#Bearer)
+
+### HTTP request headers
+
+- **Content-Type**: Not defined
+- **Accept**: application/json
+
+### HTTP response details
+
+| Status code |      Description      | Response headers |
+|-------------|-----------------------|------------------|
+| **204**     | No Content            | -                |
+| **404**     | Not Found             | -                |
+| **500**     | Internal Server Error | -                |
+
+<a name="modifyCollaborator"></a>
+
+# **modifyCollaborator**
+
+> modifyCollaborator(createCollaboratorDto)
+
+Adds a new collaborator to a project or modifies the permission level of an existing collaborator.&lt;br/&gt;&lt;br/&gt; &lt;strong&gt;Note:&lt;/strong&gt; Only users that are part of the authorized organization (see &lt;code&gt;GET /api/v1/info&lt;/code&gt;) &lt;i&gt;and&lt;/i&gt; logged in to Web Modeler at least once can be added to a project.
```

Comment:
For some reason the html syntax doesn't render properly. Is this intentional? I think we might want to find the appropriate markdown syntax

Reply:
This is how it gets generated by `openapi-generator-maven-plugin` - for the other files I manually fixed. the broken HTML tags, I simply forgot it here.

We might want to look at how to fix this in a sustainable way in the future.